src/components/layout/header/account-switcher.tsx

```diff
@@ -39,6 +41,17 @@ const tabs_labels = {
 
 const RenderAccountItems = ({ isVirtual, modifiedAccountList, switchAccount }: TAccountSwitcherProps) => {
     const { client } = useStore();
+    const account_data = useRef();
+
+    const fetchAccountSwitcherData = useCallback(async () => {
+        if (account_data.current || !client?.loginid || !modifiedAccountList) return;
+        const account_info = await checkSwitcherType(modifiedAccountList, client.loginid, isVirtual);
+        if (account_data) account_data.current = account_info;
```

Comment:
@rupato-deriv is this logic necessary or we can just put
```suggestion
        account_data.current = account_info;
```